### PR TITLE
(FRO-35): developed the about us page on zurichat

### PR DESCRIPTION
**Pull request by CyberGeni**
# Fixes Issue/Linear Ticket
Resolves ticket [FRO-35](https://linear.app/mechanic-team-prybar/issue/FRO-35/about-us)

**My PR Fixes FRO-35**

# Changes proposed

## What were you told to do?

I was assigned to work on linear ticket fro-35-about-us which involves developing the about us page
## What did you do?

I developed the about page according to a new design already vetted and reviewed. 
I adapted to using CSS Modules for page styling
I tried to ensure responsiveness across multiple screens
I used reusable components to make the code less bulky

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos
image

### CODE REVIEW

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/pages/static/about/index.jsx [hunk 1]
```diff
@@ -60,26 +110,80 @@ const About = () => {
...
             <p>
-              {t("about.whoWeAre.description")} <br />
+              <span className={` ${styles.span}`}>ZuriChat</span>{" "}
+              {t("about.whatWeDo.description")} <br />
             </p>
-            <p>
+            {/* <p>
               <span className={` ${styles.span}`}>ZuriChat</span>{" "}
               {t("about.whoWeAre.description_")}
+            </p> */}
```
[the remark this answers is not in the document]
Comment (CyberGeni) [reply packages/main/src/pages/static/about/index.jsx]: done

**Review by JulianaSau — APPROVED** [review]

**Comment by JulianaSau** [comment]
Hi @CyberGeni, please resolve your conflicts so that I can merge

**Comment by CyberGeni** [comment]
@JulianaSau i've fixed the merge conflict

**Comment by billmal071** [comment]
You have more conflicts.

File: packages/ui/src/message-pane-input/MessagePaneInput.jsx [hunk 2]
```diff
@@ -195,154 +193,46 @@ const MessagePaneInput = ({ onSendMessage, users, onAttachFile }) => {
...
-
   return (
     <Wrapper>
       <InputWrapper>
-        {preview.length ? <PreviewItem /> : null}
         {preview ? (
           <Preview>
-            <PreviewFile />
-
+            <img src={preview} alt="Image Preview" />
```
Comment (ickynavigator) [inline packages/ui/src/message-pane-input/MessagePaneInput.jsx:220]: what happens if it's not an image?

**Review by ickynavigator — CHANGES_REQUESTED** [review]
I am closing this PR because there are way too many things done wrongly

**Comment by CyberGeni** [comment]
@ickynavigator 99% of these changes here weren't made by me, they're from other PRs I pulled. I'm only responsible for the changes made to the About us page, that's the feature I was assigned to.

**Comment by JulianaSau** [comment]
@CyberGeni make a new or that contains only the changes you made

**Pull request closed without merging**